core/providers/LoggerServiceProvider.php

```diff
@@ -2,9 +2,8 @@
 
 namespace ImpressCMS\Core\Providers;
 
-use League\Container\ServiceProvider\AbstractServiceProvider;
 use icms_core_Logger;
-use icms_Event;
+use League\Container\ServiceProvider\AbstractServiceProvider;
 use League\Container\ServiceProvider\BootableServiceProviderInterface;
 
 /**
```

Comment:
looks like this is not related to aura session, but t something else?

Reply:
These lines where changed because I use auto cleanup for namespaces in PHPStorm for class definition. So, if I remove elsewhere in same class requirement for using some classes these will be automatically removed by IDE. It helps to keep code a bit cleaner.